### PR TITLE
[View Payments] Upgrade web components to their V3 versions

### PR DESCRIPTION
**Pull request by jerekshoe**
## Summary
Upgrading all of the web components in the View Payments application to use their V3 versions. Also removing `uswds="false"` on web components that don't have a V3 version currently

## Related issue(s)
- _Link to ticket created in va.gov-team repo_
department-of-veterans-affairs/va.gov-team#76127

### `va-alert`
|         | Before | After |
| ------- | ------ | ----- |
| Desktop | va-alert-before | va-alert-after |

### `va-table` / `va-pagination`
|         | Before | After |
| ------- | ------ | ----- |
| Desktop | va-table-before | va-table-after |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx [hunk 1]
```diff
@@ -114,7 +114,11 @@ class ViewPaymentsLists extends Component {
       const alertContent = isClientError(this.props.error.code)
         ? NoPaymentsContent
         : ServerErrorAlertContent;
-      content = <va-alert status={status}>{alertContent}</va-alert>;
+      content = (
+        <va-alert status={status} uswds>
+          {alertContent}
+        </va-alert>
+      );
```
Comment (pmclaren19) [inline src/applications/disability-benefits/view-payments/components/view-payments-lists/ViewPaymentsLists.jsx:121]: Line 128 also has a va-alert so we should update that as well!

**Review by micahchiang — DISMISSED** [review]

**Review by pmclaren19 — APPROVED** [review]
Thanks for updating! LGTM :)

**Review by pmclaren19 — CHANGES_REQUESTED** [review]
@jerekshoe I think we actually can remove this prop entirely? https://dsva.slack.com/archives/C055573C508/p1708110642607439 And by default it will be set to true.

**Comment by pmclaren19** [comment]
> @jerekshoe I think we actually can remove this prop entirely? https://dsva.slack.com/archives/C055573C508/p1708110642607439 And by default it will be set to true.

Talked with the Team this morning in stand up and this ticket is fine to merge, @jerekshoe will just want to create another ticket for the work to remove the `uswds=true` when [this](https://github.com/department-of-veterans-affairs/component-library/pull/1043) pr is merged.

**Review by pmclaren19 — APPROVED** [review]

**Pull request merged**